app/experimenter/projects/forms.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.utils.text import slugify
+
+from experimenter.projects.models import Project
+
+
+class ProjectForm(forms.ModelForm):
+    name = forms.CharField(
+        widget=forms.TextInput(attrs={'class': 'form-control'}))
+    slug = forms.CharField(required=False)
+
+    class Meta:
+        model = Project
+        fields = ('name', 'slug', 'image')
```

Comment:
Where's `image` defined? Do you need `slug` here if you don't show it in the template?

Reply:
Image is defined on the model, you don't need to explicitly declare it as a form field, the other fields are declared to override some behaviour.

Slug has to be here becuase the form actually manages that field, it's just that it's hidden from the user and auto populated by name.  If it's not listed here you can't set a value and write it to the model.